### ISSUE TITLE
Package Error

### ISSUE THREAD

**Issue by fabrizioleone** [nebulae-co/colmaps#3]
Hello, 

I am making use of your (very useful) package. However, it seems that something is wrong with it. 
If I type 

`colmap(municipios, subset(homicidios, año == 2000), var = "tasa")`

I get the following error:

> Theme element panel.border missing
Theme element axis.line.x.bottom missing
Theme element axis.ticks.x.bottom missing
Theme element axis.line.y.left missing
Theme element axis.ticks.y.left missing
Error in width_cm(guide$barwidth %||% theme$legend.key.width) : 
  Unknown input

It looks like there is an issue with ggmap. See [here](https://github.com/dkahle/ggmap/issues/228).

Would you be so kind to check if you can solve this issue? I guess many users will appreciate it. 

Thank you.

**Comment by juribep5** [comment]
Yes, I had the same problem. Were you able to fix it or did you use other tool to map?

**Comment by fabrizioleone** [comment]
Indeed I haven't found yet a way around this issue. Did you maybe do it?
